ui/src/scss/_patterns_icons.scss

```diff
@@ -0,0 +1,49 @@
+.p-icon--chevron {
```

Comment:
Vanilla changed it to `p-icon--contextual-menu` so you can use that and remove this class